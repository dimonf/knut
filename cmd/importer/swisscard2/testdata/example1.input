Transaktionsdatum,Beschreibung,Händler,Kartennummer,Währung,Betrag,Fremdwährung,Betrag in Fremdwährung,Debit/Kredit,Status,Händlerkategorie,Registrierte Kategorie
"06.07.2024","a","aa","11","CHF","72.60","","","Belastung","Gebucht","Familie & Haushalt","FAMILY CLOTHING STORES"
"06.07.2024","b","bb","22","CHF","6.00","","","Belastung","Gebucht","Auto","AUTOMOBILE TRUCK DEALERS, SALES, SERVICE"
"06.07.2024","c","cc","33","CHF","80.70","","","Belastung","Gebucht","Lebensmittel","DEPARTMENT STORES"
"06.07.2024","d","dd","44","CHF","4.95","","","Belastung","Gebucht","Lebensmittel","DEPARTMENT STORES"
"05.07.2024","e","ee","55","CHF","36.00","","","Belastung","Gebucht","Gesundheit und Schönheit","DRUG STORES and Pharmacies"
