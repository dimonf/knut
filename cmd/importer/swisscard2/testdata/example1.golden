2024-07-05 "e / ee / Gesundheit und Sch√∂nheit / 55 / DRUG STORES and Pharmacies / Belastung"
Liabilities:CreditCard Expenses:TBD                   36 CHF

2024-07-06 "a / aa / Familie & Haushalt / 11 / FAMILY CLOTHING STORES / Belastung"
Liabilities:CreditCard Expenses:TBD                 72.6 CHF

2024-07-06 "b / bb / Auto / 22 / AUTOMOBILE TRUCK DEALERS, SALES, SERVICE / Belastung"
Liabilities:CreditCard Expenses:TBD                    6 CHF

2024-07-06 "c / cc / Lebensmittel / 33 / DEPARTMENT STORES / Belastung"
Liabilities:CreditCard Expenses:TBD                 80.7 CHF

2024-07-06 "d / dd / Lebensmittel / 44 / DEPARTMENT STORES / Belastung"
Liabilities:CreditCard Expenses:TBD                 4.95 CHF

